### ISSUE TITLE
A11y: Incorrect aria-role on psammead-navigation

### ISSUE THREAD

**Issue by 12** [BBC-archive/psammead#2472]
**Describe the bug**
When running the axe accessibility chrome plugin on https://www.test.bbc.co.uk/news/articles/c3wmq4d1y3wo, a violation was picked up regarding the `Home` item in the navigation. `text` is not a valid `aria-role`.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to an article with navigation
2. Open Chrome devtools
3. Run axe
4. See accessibility violation

**Expected behaviour**
When running the axe accessibility tool, I should see no violations.

**Screenshots**
![image](https://user-images.githubusercontent.com/7690572/67392770-0c7f9500-f599-11e9-9920-3b172e19923a.png)

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
See valid roles here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

**Comment by DenisHdz** [comment]
This is expected. The `role="text"` is not a valid aria according to the specs but improves the screen reader experience. @greenc05 has asked Lighthouse/Axe to add an exception for this https://github.com/dequelabs/axe-core/issues/1597

**Comment by greenc05** [comment]
@12 As a side note, 'Home' should only be the current page/selected page if it's the home page
